k8s/upgrades/0.8.0-0.8.1/cstor_pool_post_upgrade.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+function usage() {
+ echo
+ echo "Usage:"
+ echo
+ echo "$0 <spc-name> <openebs-namespace>"
+ echo
+ echo "  <spc-name> Get the SPC name using: kubectl get spc"
+ echo "  <openebs-namespace> Get the namespace where pool pods"
+ echo "    corresponding to SPC are deployed"
+ exit 1
+}
+
+if [ "$#" -ne 2 ]; then
+ usage
+fi
+
+spc=$1
+ns=$2
+retry=false
+
+## Fetching the pod names corresponding to spc
+running_pool_pods=$(kubectl get po -n $ns -l app=cstor-pool,openebs.io/storage-pool-claim=$spc \
+                 -o jsonpath='{range .items[*]}{@.metadata.name}:{end}')
+
+## Setting the quorum enabled in pool pods ###
+for pool_pod in `echo $running_pool_pods | tr ":" " "`; do
+
+pool_name=""
+cstor_uid=""
+cstor_uid="$(kubectl get pod $pool_pod -n $ns \
+            -o jsonpath="{.spec.containers[*].env[?(@.name=='OPENEBS_IO_CSTOR_ID')].value}" | awk '{print $1}')"
+
+pool_name="cstor-$cstor_uid"
+quorum_set=$(kubectl exec $pool_pod -n $ns -c cstor-pool-mgmt -- zfs set quorum=on $pool_name)
+rc=$?
+if [[ ($rc -ne 0) ]]; then
+ echo "Actual error msg: failed to set quorum=on at $pool_name"
+ retry=true
+fi
+
+output=$(kubectl exec $pool_pod -n $ns -c cstor-pool-mgmt -- zfs get quorum)
+rc=$?
+if [ $rc -ne 0 ]; then
+ echo "ERROR: while executing zfs get quorum for pool $pool_name, error: $rc"
+ retry=true
+fi
+
+no_of_non_quorum_vol=$(echo $output | grep -wo off | wc -l)
+if [ $no_of_non_quorum_vol -ne 0 ]; then
+ echo "Failed to get quorum values from pool $pool_name, error: $rc"
```

Comment:
print no_of_non_quorum_vol value also

Reply:
Change error message as "Few(%d) of quorum values are having inappropriate values for quorum"